packages/alfa-aria/src/node/element.ts

```diff
@@ -115,6 +119,65 @@ export class Element extends Node<"element"> {
       ...this._children.map((child) => String.indent(child.toString())),
     ].join("\n");
   }
+
+  private static allowedAttributesForInputType(
+    inputType: InputType
+  ): readonly Attribute.Name[] {
```

Comment:
```suggestion
  ): ReadonlyArray<Attribute.Name> {
```
We usually use the `Array<T>` type syntax rather than `T[]` 🤷